src/generator/AutoRest.Go/Model/CodeModelGo.cs

```diff
@@ -228,5 +236,33 @@ public IEnumerable<MethodGo> ClientMethods
                 return Methods.Cast<MethodGo>().Where(m => string.IsNullOrEmpty(m.MethodGroup.Name));
             }
         }
+
+        /// FormatVersion normalizes a version string into a SemVer if it resembles one. Otherwise,
+        /// it returns the original string unmodified. If version is empty or only comprised of
+        /// whitespace, 
+        public static string FormatVersion(string version) {
```

Comment:
I'd be willing to throw the FormatVersion function back in CodeNamerGo.go, but I thought it made sense to put it with the only place it is used.

Just let me know!